Examples/Examples/FeatureFormExampleView.swift

```diff
@@ -311,7 +311,7 @@ class Model: ObservableObject {
         do {
             if let serviceInfo = database.serviceInfo, serviceInfo.canUseServiceGeodatabaseApplyEdits {
                 let featureTableEditResults = try await database.applyEdits()
-                resultErrors = featureTableEditResults.flatMap { $0.editResults.errors }
+                resultErrors = featureTableEditResults.flatMap(\.editResults.errors)
```

Comment:
Can you please also update the tutorial files [here](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/blob/8c2225ca27bc6d4d6658ec18f6f09194851e1877/Sources/ArcGISToolkit/Documentation.docc/Resources/FeatureFormView/FeatureFormViewStep10.swift#L154), starting from the file for step 10 onwards? Thanks!